group/urls.py

```diff
@@ -8,4 +8,5 @@
     path('detail/<int:group_id>/', GroupDetailApiView.as_view()),
     path('user/<int:user_id>/', GroupDetailApiView.as_view()),
     path('message', MessageAPIView.as_view()),
+    path('messages', MessageAPIView.as_view()),
```

Comment:
I would expect this url to be the same as the one on line 10, with the difference being a primary key identifier, say for the user making the get request for their messages . i.e message/<user_id>
Please refer to this resource for RESTFUL API best practices https://www.sitepoint.com/build-restful-apis-best-practices/